File: FoodieWebAPI/Foodie.ManagementAPI/ResponseDto/Token.cs
namespace Foodie.ManagementAPI.ResponseDto
{
    public class Token
    {
        public string TokenString { get; set; }
        public DateTime Expiration { get; set; }
        public UserResponse User { get; set; }

        public Token(string tokenString, DateTime expiration, UserResponse user)
        {
            TokenString = tokenString;
            Expiration = expiration;
            User = user;
        }
    }
}

File: FoodieAppAPI/app/src/main/java/com/example/foodie/models/Token.java
package com.example.foodie.models;

public class Token {
    private String tokenString;
    private String expiration;

    public Token() {
    }

    public String getTokenString() {
        return tokenString;
    }

    public void setTokenString(String tokenString) {
        this.tokenString = tokenString;
    }

    public String getExpiration() {
        return expiration;
    }

    public void setExpiration(String expiration) {
        this.expiration = expiration;
    }

    @Override
    public String toString() {
        return "Token{" +
                "tokenString='" + tokenString + '\'' +
                ", expiration='" + expiration + '\'' +
                '}';
    }
}
